utils/helpers.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { Icon } from 'native-base';
+
+const styles = StyleSheet.create({
+  icon: {
+    fontSize: 16,
+  },
+});
+
+export const alertTypes = {
+  WATEROUTAGE: {
+    name: 'Water Outage',
+    color: 'orange',
+    iconName: 'water-off',
+    iconType: 'MaterialCommunityIcons',
+  },
+  TRAFFICDISRUPTIONS: {
+    name: 'Traffic Distruptions',
+    color: '#BF5700',
+    iconName: 'traffic-cone',
+    iconType: 'Entypo',
+  },
+  DONOTDRINK: {
+    name: 'Do Not Drink',
+    color: 'red',
+    iconName: 'circle-slash',
+    iconType: 'Octicons',
+  },
+  BOILWATERNOTICE: {
+    name: 'Boil Water Notice',
+    color: 'red',
+    iconName: 'kettle',
+    iconType: 'MaterialCommunityIcons',
+  },
+};
+
+/**
+ * @function groupBy
+ * groups Array by Key into object
+ */
+export function groupBy(xs, key) {
+  return xs.reduce((rvIn, x) => {
+    const rv = rvIn;
+    (rv[x[key]] = rv[x[key]] || []).push(x);
+    return rv;
+  }, {});
+}
+
+/**
+ * @function getNoticeType
+ * return object with notice name, style and icon
+ */
+export function getNoticeType(type) {
+  if (alertTypes[type]) {
+    return {
+      name: alertTypes[type].name,
+      getIcon: () => (
+        <Icon
+          name={alertTypes[type].iconName}
+          type={alertTypes[type].iconType}
+          style={[styles.icon, { color: alertTypes[type].color }]}
+        />
+      ),
+      getStyle: () => ({
+        color: alertTypes[type].color,
+      }),
+    };
+  }
+  return {
+    name: type && type.length ? type : 'Unknown Notice Type',
+    getIcon: () => (
+      <Icon name="alert" style={[styles.icon, { color: 'black' }]} />
+    ),
+    getStyle: () => ({
+      color: 'black',
+    }),
+  };
+}
+
+/**
+ * @function formatDate
+ * formatsDate in irish format
+ */
+export function formatDate(date) {
+  return `${date.getDate()}/${date.getMonth() + 1}/${date.getYear() - 100}`;
```

Comment:
why do you have to use date.getYear() -100?

Reply:
"For years greater than or equal to 2000, the value returned by getYear() is 100 or greater. For example, if the year is 2026, getYear() returns 126." - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getYear

Reply:
The documentation actually says that this function should no longer be used and to use - date.getFullYear() instead. Can you change it to that one please? We can then see what looks better the four digit year or two digit year.

Reply:
Good catch, I'll fix that for next pull request with items.